src/util/useCheckCanEdit.ts

```diff
@@ -1,9 +1,17 @@
 import useOktaTokens from "../hooks/useOktaTokens";
 import { Acl } from "@madie/madie-models/dist/Measure";
 
-const useCheckUserCanEdit = (createdBy: string, acls: Array<Acl>): boolean => {
+const useCheckUserCanEdit = (
+  createdBy: string,
+  acls: Array<Acl>,
+  version: string = "0.0.000"
+): boolean => {
   const { getUserName } = useOktaTokens();
   const userName = getUserName();
+  // versioned measures are always uneditable.
+  if (version !== "0.0.000") {
+    return false;
```

Comment:
I believe it is ideal to use the **_Measure.MeasureMetadata.draft_** flag in order to determine the state of a measure.

Reply:
got it, will change this thank you